src/library/Uno.Themes.WinUI.Markup/Theme/ToggleSwitch.cs

```diff
@@ -0,0 +1,206 @@
+﻿using System;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Media;
+using Uno.Extensions.Markup.Internals;
+
+namespace Uno.Themes.Markup
+{
+	public static partial class Theme
+	{
+		public static class ToggleSwitch
+		{
+			public static class Resources
+			{
+				public static class OuterBorder
+				{
+					public static class Fill
+					{
+						[ResourceKeyDefinition(typeof(Brush), "ToggleSwitchOuterBorderFill")]
+						public static ResourceValue<Brush> Default => new("ToggleSwitchOuterBorderFill", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "ToggleSwitchOffOuterBorderFill")]
+						public static ResourceValue<Brush> Off => new("ToggleSwitchOffOuterBorderFill", true);
+
+					}
+
+					public static class Stroke
+					{
+						[ResourceKeyDefinition(typeof(Brush), "ToggleSwitchOuterBorderStroke")]
+						public static ResourceValue<Brush> Default => new("ToggleSwitchOuterBorderStroke", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "ToggleSwitchOffOuterBorderStroke")]
+						public static ResourceValue<Brush> Off => new("ToggleSwitchOffOuterBorderStroke", true);
+					}
+				}
+
+				public static class Knob
```

Comment:
I'm wondering if we should make two subclasses of `Knob`, `On` and `Off`, instead of having every property have a bunch of `OnX` and `OffX` visual states. @kazo0

Reply:
Yeah I like that: `ToggleSwitch.Resources.Knob.On.Fill.Default/PointerOver/Focused/etc.`

Reply:
On second thought, following the updated specs [here](https://github.com/unoplatform/uno.themes-private/issues/16#issue-1909235521)

It should be `ToggleSwitch.Resources.Knob.OnFill.Default/PointerOver/Focused/etc.` and `ToggleSwitch.Resources.Knob.OffFill.Default/PointerOver/Focused/etc.`